src/js/media-theme-element.js

```diff
@@ -24,6 +24,16 @@ prependTemplate.innerHTML = `
       width: 100%;
       height: 100%;
     }
+
+    media-controller:not(:is([mediacaptionslist], [mediasubtitleslist])) media-captions-selectmenu,
+    media-captions-button:not(:is([mediacaptionslist], [mediasubtitleslist])),
```

Comment:
though(non-blocking): Should we use `:where` instead to limit the selector specificity?

Reply:
yes, that's a good idea. will update